src/include/trusted_ca_keys.h

```diff
@@ -0,0 +1,45 @@
+// Copyright 2023 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef TRUSTED_CA_KEYS_H_
+#define TRUSTED_CA_KEYS_H_
+
+#include <vector>
+#include <fstream>
+#include <string>
+
+using std::string;
+using std::vector;
+using std::ofstream;
+
+// A default API endpoint
+const char *mds_url_def = "http://169.254.169.254/computeMetadata/v1/oslogin/certificates";
```

Comment:
`kMetadataServerUrl` has a different value.  it is not equal to `mds_url_def`.

"The code base" has another use case. It _dynamically_ at runtime concatenates strings by adding a variable part to the static prefix in the stringstreams and then converts it into strings. Why on Earth I need to do this for the _always_ static URL? It is not changing. There are no dynamic parts. It always has the same value.

If you suggest concatenate it at runtime, something like 

```
std::stringstream url;
url << kMetadataServerUrl << "certificates";
// ...and then finally get that string by calling   url.str()
```
instead of a constant char array that initializes a string, sorry, I disagree.
I see a perfect use case for the static char array here.